src/components/pages/mainPage/leftSideBar/index.js

```diff
@@ -1,11 +1,13 @@
 import React, { Component } from 'react';
 
 import MiniProfile from './miniProfile';
+import AccessProfile from './accessBar';
 
 export default class LeftSideBar extends Component {
 	render() {
 		return (
-			<MiniProfile />
+			// <MiniProfile />
```

Comment:
This should be a conditional
Pseudocode: `(userIsLoggedIn) ? <MiniProfile /> : <AccessProfile />`